server/models/Request.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+const { addressSchema } = require("./Address");
+
+const requestSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  sitterId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  start: {
+    type: Date,
+    required: true,
+  },
+  end: {
+    type: Date,
+    required: true,
+  },
+  accepted: {
+    type: Boolean,
+    required: true,
+    default: false,
+  },
+  declined: {
+    type: Boolean,
+    required: true,
+    default: false,
+  },
+  paid: {
+    type: Boolean,
+    required: true,
+    default: false,
+  },
+  address: addressSchema,
```

Comment:
I like the idea of having an address but the schema is not what you would expect the address to be. No street numbers or anything? 

Could we make that schema an actual address schema. I'm not worried about pickup or drop-off particularly.

Reply:
Also, I think we can just have the customer enter an address and then we can just input that in there automatically.

Reply:
That's... I'm not sure why I didn't think of it that way, lol. I envisioned the address just being a string like, "123 Test St, Calgary, AB, H0H 0H0". But yeah, that's a much better way of doing it. I'll get it changed around!

Reply:
Yeah, I think ideally we just have them enter it in the settings then we just use that (maybe in the future can have multiple addresses)